plugins/doc_fragments/vmware.py

```diff
@@ -46,6 +46,16 @@ class ModuleDocFragment(object):
       - If the value is not specified in the task, the value of environment variable E(VMWARE_PORT) will be used instead.
       type: int
       default: 443
+    datacenter:
+      description:
```

Comment:
Does it a required parameter?

Reply:
for this module yes it is because we need to get the absolute folder path in order to save the template. And the folder path is <datacenter>/vms/whatever

Reply:
I found it later... Resolving this comment